deployments/apiv2/services/users/src/main.py

```diff
@@ -732,7 +748,7 @@ async def register_user(
                         email=email,
                         url=f"{DASHBOARD_URL}",
                         subject="Your User account has been created",
-                        template="registration_sso.html"
+                        template="registration_sso.html",
```

Comment:
is there a linting step that I'm missing?

Reply:
I think we forgot to mention how to set up pre-commit, it will handle linting, formatting, etc for us each time we commit. I don't remember the pre-commit set up steps exactly, but I think you can just pip install pre-commit globally and the once in the root dir of any repo run `pre-commit install` then every `git commit` will also run pre-commit. The .pre-commit-config.yaml files in our repos have the min pre-commit versions required